Assets/EmuHawkMono.sh

```diff
@@ -8,11 +8,14 @@ libpath=""
 if [ "$(command -v lsb_release)" ]; then
 	case "$(lsb_release -i | cut -c17- | tr -d "\n")" in
 		"Arch"|"ManjaroLinux") libpath="/usr/lib/wine";;
-		"Debian"|"Ubuntu"|"LinuxMint") libpath="/usr/lib/x86_64-linux-gnu/wine";;
+		"Debian"|"LinuxMint") libpath="/usr/lib/x86_64-linux-gnu/wine";;
+		"Ubuntu") libpath="/usr/lib/x86_64-linux-gnu/wine"; export MONO_WINFORMS_XIM_STYLE=disabled;; # see https://bugzilla.xamarin.com/show_bug.cgi?id=28047#c9
 	esac
+else
+	printf "Distro does not provide LSB release info API! (You've met with a terrible fate, haven't you?)\n"
 fi
 if [ -z "$libpath" ]; then
 	printf "%s\n" "Unknown distro, assuming WINE library location is /usr/lib/wine..."
```

Comment:
Is wine still required as a dep? My memory is hazy but I thought we covered that off possibly?

Reply:
not in this branch